hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3271,9 +3271,16 @@ func (r *HostedClusterReconciler) reconcileNetworkPolicies(ctx context.Context,
 	}
 
 	// Reconcile KAS Network Policy
+	var managementClusterNetwork *configv1.Network
+	if r.ManagementClusterCapabilities.Has(capabilities.CapabilityNetworks) {
+		managementClusterNetwork = &configv1.Network{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
```

Comment:
dont we require to grant additional rbac for this?

Reply:
HO already has a ClusterRole that gives it `get` on everything in `config.openshift.io`